packages/core/stories/patterns/metric/metric.stories.tsx

```diff
@@ -0,0 +1,172 @@
+import {
+  StackLayout,
+  Text,
+  Display1,
+  Display2,
+  Display3,
+  Link,
+} from "@salt-ds/core";
+import { ArrowUpIcon, ArrowDownIcon } from "@salt-ds/icons";
+import { Meta } from "@storybook/react";
+
+export default {
+  title: "Patterns/Metric",
+} as Meta;
+
+export const Metric = () => {
+  return (
+    <StackLayout gap={0}>
+      <Text>
+        <strong>Revenue YTD</strong>
+      </Text>
+      <Display1>$801.9B</Display1>
+    </StackLayout>
+  );
+};
+
+export const HorizontalMetric = () => {
+  return (
+    <StackLayout direction="row" gap={1}>
+      <Text>
+        <strong>Revenue YTD</strong>
+      </Text>
+      <Display1 style={{ lineHeight: "0.8" }}>$801.9B</Display1>
```

Comment:
This doesn't match the spec cc @yunjungyeh. Talking to Yun the text should have margin/padding applied at spacing-100 rather than this alignment